vimoutliner/plugin/votl_smart_paste.vim

```diff
@@ -4,10 +4,10 @@
 " middle of a branch.
 " Added 2011-03-01(JB): This script will now also copy an outline correctly by
 " using \\y, copy to the register with \\r, cut an outline by using \\d, and
-" paste from the register using \\b. 
+" paste from the register using \\b.
 " http://www.lists.vimoutliner.org/pipermail/vimoutliner/2008-October/002366.html
 
-map <buffer>p :call VOput()<cr>
```

Comment:
That is not a typo.
Look up   `:help c_ctrl-u`
Ponder what the code would do without that if the hapless user were to try something like: `3"*p`